src/main/java/com/globallogic/amcr/controller/contactcomponent/FeedbackController.java

```diff
@@ -24,17 +25,14 @@
 /**
  * endpoint for feedback upload and download
  */
-
 @RestController
 @RequestMapping("/feedback")
-@CrossOrigin(origins = "*")
+@CrossOrigin(origins = {"http://localhost:3000", "http://ec-acad-elb-fe-2f7bfb1526a617ef.elb.eu-west-2.amazonaws.com:3000"})
```

Comment:
Okay sounds good. I was thinking that to be fair, it would be good to have different origins for different environments. I will look into WebMvcConfigurer.